test/linalg/lu.jl

```diff
@@ -167,7 +173,9 @@ end
 ## Testing Rational{BigInt} and BigFloat version
 nHilbert = 50
 H = Rational{BigInt}[1//(i+j-1) for i = 1:nHilbert,j = 1:nHilbert]
-Hinv = Rational{BigInt}[(-1)^(i+j)*(i+j-1)*binomial(nHilbert+i-1,nHilbert-j)*binomial(nHilbert+j-1,nHilbert-i)*binomial(i+j-2,i-1)^2 for i = big(1):nHilbert,j=big(1):nHilbert]
+Hinv = Rational{BigInt}[(-1)^(i+j)*(i+j-1)*binomial(nHilbert+i-1,nHilbert-j)*
+    binomial(nHilbert+j-1,nHilbert-i)*binomial(i+j-2,i-1)^2
+    for i = big(1):nHilbert,j=big(1):nHilbert]
```

Comment:
I'm not much of a fan of horizontal-alignment games like that, I'd rather just apply a simple set of rules that we could conceivably automate some day.